src/Data/List/Relation/Binary/Sublist/Setoid.agda

```diff
@@ -79,3 +79,80 @@ open HeterogeneousProperties.Antisymmetry {R = _≈_} {S = _≈_} (λ x≈y _ 
 ⊆-poset = record
   { isPartialOrder = ⊆-isPartialOrder
   }
+
+------------------------------------------------------------------------
+-- Weak pushout
```

Comment:
Do you think this content should go in `Data.List.Relation.Binary.Sublist.Setoid.Categorical`?

Reply:
Currently, the `Category` tree defines functional control structures such as `RawMonad` and relatives. (Imho, this tree would be better named `Control` like in Haskell to free `Category` for proper category theory; but this would be a too invasive change.)
The stuff under `Categorical` seems to confine itself to define Monad instances etc.
Thus, I am not sure whether such a `Categorical` module would be similar enough to the existing ones. Dunno. @gallais, what would you think?

Reply:
> Imho, this tree would be better named Control like in Haskell to free Category for proper category theory; but this would be a too invasive change.

Hmm I guess we could change it to `Control` if we kept the old modules around for backwards compatibility... I agree `Categorical` was a bit of a weird naming choice on my part. The beauty of hindsight :smile:

Reply:
`Categorical` makes sense given the pre-existing `Category.(Functor/Applicative/Monad)` modules.

Reply:
This is why, in part, the current `agda-categories` uses `Categories` as the top-level name space, so that it can be used alongside `stdlib` with no conflicts. If that name should be changed, the sooner we do that, the better.

Reply:
So I think using `Categories` as a top-level subsection is fine. Then, just like `Relation`, different parts of the library can have a `Categorical` folder where they use those definitions.

Reply:
Concerning this PR, I propose to leave the `RawPushout` here until the `Category` tree goes beyond the category of `Set` and `->`.  Then, the definition of `RawPushout` can be moved to the `Category` tree and the `⊆-pushoutˡ` to a `Categorical` submodule.